### ISSUE TITLE
Taxi command taxing to wrong runway (Dubai)

### ISSUE THREAD

**Issue by Dropbear67** [openscope/openscope#1119]
Browser: Safari

Airport: Dubai

Approximate play time before issue: ANY

#### Issue Description
_please include any console errors and/or screenshots_ 
issuing the commands TAXI CAF sometimes reports that the runway 31L is not an appropriate runway for the SID (or similar) and then I have to manually specify TAXI 31R CAF

#### Steps to reproduce
- issue command TAXI CAF on a SID that requires a 31R runway

**Comment by Marcel510** [comment]
The default departure runway for OMDB is runway 30R, that's the runway that should be when you only issue "taxi" without a specific runway. Can you confirm that aircraft also taxi to 30L when only using "taxi"?
There shouldn't be any routing problems if you use 30R for departures and 30L for landings.

**Comment by Dropbear67** [comment]
My apologies. The airport is Abu Dhabi not Dubai... and yes the same holds.. a TAXI without a runway will go to 31L

**Comment by felixscheffer** [comment]
I can't see a bug here. Sending an aircraft to a runway that is not valid for its assigned SID is not a bug but a mistake by the controller.

**Comment by Marcel510** [comment]
@felixscheffer 
Well, it is a mistake by the controller but its hard not to do the mistake since you would have to know all spawnpatterns and their according runways. Therefore I've already suggested to add the assigned runway of an aircraft to the flight strips (see #1003) and maybe change to function of the taxi command: When issuing "taxi" without adding a specfic runway the aircraft should taxi to it's assigned runway instead of the default departure runway.

If we don't have that we should, in my opinion, at least elucidate in the airport guide which runways are used for which SIDs. (f.e. [KATL airport guide](https://github.com/openscope/openscope/blob/develop/documentation/airport-guides/katl.md#runway-configuration))

**Comment by felixscheffer** [comment]
> but its hard not to do the mistake since you would have to know all spawnpatterns and their according runways

Absolutely. But that's the way real life ATC works. You need to know your SIDs and STARs and restrictions and what not. 

> When issuing "taxi" without adding a specfic runway the aircraft should taxi to it's assigned runway instead of the default departure runway.

I have never been a big fan of "taxi without runway". It's completely unrealistic. Personally, I am in favor of removing "taxi without runway" (and make the runway argument mandatory) instead of improving it.

> at least elucidate in the airport guide which runways are used for which SIDs.

Also, the airport guides should be available from within openScope or at least there should be a link to each airport guide in openScope.

**Comment by Marcel510** [comment]
> Absolutely. But that's the way real life ATC works. You need to know your SIDs and STARs and restrictions and what not.

@felixscheffer 
And what about adding the departure runway to the strips?
In my opinion that's not unrealistic that might differ from facility to facility but I'm pretty sure some facilities have that:
Have a look at this document http://tfmlearning.faa.gov/Publications/atpubs/ATC/atc0203.html,
2-3-4. TERMINAL DATA ENTRIES, b. Departures and then block 10-18:
![](https://user-images.githubusercontent.com/24833249/46956563-e970fa00-d095-11e8-83e0-3296c8415f48.png)

**Comment by indianbhaji** [comment]
Technically all this taxiing stuff should not be involved in a simulator where we’re simulating the approach and centre positions. @erikquinn specified an interest in removing takeoff, taxi and landing clearances completely

**Comment by felixscheffer** [comment]
> Isn't in reality the preferred runway determined by an ATC and airline files a flight plan according to it?

Correct.

> And what about adding the departure runway to the strips?

I love the idea (see #1003)

> Technically all this taxiing stuff should not be involved in a simulator where we’re simulating the approach and centre positions.

I would love to see taxiways and ground control in openScope once we support different positions / sectors. Just dreaming... :see_no_evil:

**Comment by erikquinn** [comment]
>> Isn't in reality the preferred runway determined by an ATC and airline files a flight plan according to it?

>Correct.

I think this is true for most of Europe, but not necessarily elsewhere. In the US, SIDs are generally not related to which runway to use. Sometimes there are different SIDs for when the airport changes direction, but looking at a flight plan would give you no inkling of what runway will be assigned. More often, the runways that aren't included in those procedures are because of some other operational reason-- such as runways that you just don't ever use for departure, or runways used only by small aircraft, etc.

> I have never been a big fan of "taxi without runway". It's completely unrealistic. Personally, I am in favor of removing "taxi without runway" (and make the runway argument mandatory) instead of improving it.

I'm surprised we haven't already done so. Created #1123. :+1:

**Comment by Dropbear67** [comment]
Good discussion all. I would be happy if the strip indicated which runway was expected for that SID, but I think it's unreasonable to expect people to know by heart what runway applies to what SID. 

I think it's also more realistic from a TRACON type point of view  not to have to taxi or clear aircraft for taxi or take off at all - thats the tower's job
